src/components/DropdownV2/DropdownV2.js

```diff
@@ -130,87 +147,115 @@ export default class DropdownV2 extends React.Component {
+      [`${prefix}--label--disabled`]: disabled,
     });
     const title = titleText ? (
-      <label htmlFor={id} className={`${prefix}--label`}>
+      <label htmlFor={id} className={titleClasses}>
         {titleText}
       </label>
     ) : null;
+    const helperClasses = cx(`${prefix}--form__helper-text`, {
+      [`${prefix}--form__helper-text--disabled`]: disabled,
+    });
     const helper = helperText ? (
-      <div className={`${prefix}--form__helper-text`}>{helperText}</div>
+      <div className={helperClasses}>{helperText}</div>
     ) : null;
+    const wrapperClasses = cx(
+      `${prefix}--dropdown__wrapper`,
+      `${prefix}--list-box__wrapper`,
+      {
+        [`${prefix}--dropdown__wrapper--inline`]: inline,
+        [`${prefix}--list-box__wrapper--inline`]: inline,
+        [`${prefix}--dropdown__wrapper--inline--invalid`]: inline && invalid,
+        [`${prefix}--list-box__wrapper--inline--invalid`]: inline && invalid,
+      }
+    );
 
     // needs to be Capitalized for react to render it correctly
     const ItemToElement = itemToElement;
     const Dropdown = (
-      <Downshift
-        id={id}
-        onChange={this.handleOnChange}
-        itemToString={itemToString}
-        defaultSelectedItem={initialSelectedItem}
-        selectedItem={selectedItem}>
-        {({
-          isOpen,
-          itemToString,
-          selectedItem,
-          highlightedIndex,
-          getRootProps,
-          getButtonProps,
-          getItemProps,
-          getLabelProps,
-        }) => (
-          <ListBox
-            type={type}
-            className={className}
-            disabled={disabled}
-            ariaLabel={ariaLabel}
-            {...getRootProps({ refKey: 'innerRef' })}>
-            <ListBox.Field {...getButtonProps({ disabled })}>
-              <span
-                className={`${prefix}--list-box__label`}
-                {...getLabelProps()}>
-                {selectedItem ? itemToString(selectedItem) : label}
-              </span>
-              <ListBox.MenuIcon isOpen={isOpen} />
-            </ListBox.Field>
-            {isOpen && (
-              <ListBox.Menu>
-                {items.map((item, index) => (
-                  <ListBox.MenuItem
-                    key={itemToString(item)}
-                    isActive={selectedItem === item}
-                    isHighlighted={
-                      highlightedIndex === index || selectedItem === item
-                    }
-                    {...getItemProps({ item, index })}>
-                    {itemToElement ? (
-                      <ItemToElement key={itemToString(item)} {...item} />
-                    ) : (
-                      itemToString(item)
-                    )}
-                  </ListBox.MenuItem>
-                ))}
-              </ListBox.Menu>
-            )}
-          </ListBox>
-        )}
-      </Downshift>
-    );
-    return title || helper ? (
-      <>
+      <div className={wrapperClasses}>
```

Comment:
Given the change here - Wanted to see it's tested in `v9`. Thanks!

Reply:
I have tested it in v9 but I will rope it off as we don't have a wrapper in v9